docs/BreakingChanges/167 WCF message security unable to use TLS1.1 and TLS1.2.md

```diff
@@ -0,0 +1,28 @@
+## 167: WCF message security unable to use TLS1.1 and TLS1.2
+
+### Scope
+Edge
+
+### Version Introduced
+4.7
+
+### Source Analyzer Status
+Planned
+
+### Change Description
+Starting in .NET Framework 4.7, customer can configure either TLS1.1 or TLS1.2 in WCF message security in addition to SSL3.0 and TLS1.0 through the Application settings. 
```

Comment:
@rpetrusha, the change is to enable the usage of TLS1.1 or TLS1.2 in WCF service with message security and client credential type Certificate. Before the fix, you are not able to use TLS1.1 or TLS1.2. If SSL3.0 and TLS1.0 are disabled in the registry, the communication will just fail. After the fix, you can specify the protocol you want to use through the following code System.Net.ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls11